include/stronk/skills/can_isnan.hpp

```diff
@@ -15,32 +15,24 @@ struct can_isnan
     constexpr auto isnan() const noexcept -> bool
     {
         static_assert(std::is_floating_point_v<typename StronkT::underlying_type>);
-        return std::isnan(static_cast<const StronkT&>(*this).template unwrap<StronkT>());
+        return twig::isnan(static_cast<const StronkT&>(*this));
```

Comment:
In here you call `twig::isnan`, but for abs you do `using twig::abs; return abs...`.

Would we want to only call `twig::abs` and `twig::isnan` from here, such that ADL only happens in those implementations?

Reply:
I only want adl in the free standing functions - I've changed it here